### PR TITLE
updates and docs

### PR DESCRIPTION
**Pull request by dtolb**
Make sure you've checked off all these things before submitting:

- [x] This pull request contains 100% test coverage.
- [x] This pull request is completely documented. All available fields are listed in the docs with description.
- [x] Each public facing function should include at least one @example in the jsdoc.
- [x] Run `npm test` before committing.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -853,6 +853,37 @@ client.Bridge.update('brg-65dhjbanasiei', {
...
+};
+
+client.Bridge.update("{bridgeId}", bridgeOptions)
+.then(function () {
+	// continue
+});
+```
+**Example**  
+```js
+// Add two calls to bridge the remove one
```
Comment (a-type) [inline docs/api.md]: > then

?

File: docs/api.md [hunk 2]
```diff
@@ -853,6 +853,37 @@ client.Bridge.update('brg-65dhjbanasiei', {
...
+var bridgeOptions = {
+	bridgeAudio : true,
+	callIds: ["{callId1","callId2"]
+};
+
+client.Bridge.update("{bridgeId}", bridgeOptions)
+.then(function () {
+	var removeCall2 = {
```
Comment (a-type) [inline docs/api.md]: Data objects with verb names irk me. This whole block would be simpler with this inlined...

```javascript
.then(client.Bridge.update("{bridgeId}", {
  callIds: ["{callId1}"]
}))
```

File: docs/api.md [hunk 3]
```diff
@@ -853,6 +853,37 @@ client.Bridge.update('brg-65dhjbanasiei', {
...
+.then(function () {
+	// continue
+});
+```
+**Example**  
+```js
+// Add two calls to bridge the remove one
+var bridgeOptions = {
+	bridgeAudio : true,
+	callIds: ["{callId1","callId2"]
```
Comment (a-type) [inline docs/api.md]: These ids look malformed.

File: test/call-test.js [hunk 4]
```diff
@@ -280,8 +282,47 @@ describe("Call API", function () {
...
+					.post("/v1/users/" + userId + "/calls/" + testCall.id, disableRecordingPayload)
+					.reply(200);
+				});
+
+				after(function () {
+					nock.cleanAll();
+					nock.enableNetConnect();
+				});
+
+				it("should enable recording on a call", function () {
```
Comment (a-type) [inline test/call-test.js]: > disable

?

**Review by a-type — CHANGES_REQUESTED** [review]

File: docs/api.md (hunk 2 above)
Comment (dtolb) [reply docs/api.md]: Shooting for verbosity over simplicity in demos. However, did update object name

File: test/call-test.js (hunk 4 above)
Comment (dtolb) [reply test/call-test.js]: updated down below

File: docs/api.md (hunk 1 above)
Comment (dtolb) [reply docs/api.md]: 👍

File: docs/api.md (hunk 3 above)
Comment (dtolb) [reply docs/api.md]: 👍

File: docs/api.md [hunk 5]
```diff
@@ -867,18 +867,18 @@ client.Bridge.update("{bridgeId}", bridgeOptions)
...
 var bridgeOptions = {
 	bridgeAudio : true,
-	callIds: ["{callId1","callId2"]
+	callIds: ["{callId1}","{callId2}"]
 };
 
 client.Bridge.update("{bridgeId}", bridgeOptions)
 .then(function () {
-	var removeCall2 = {
+	var callIdsToRemainInBridge = {
```
Comment (a-type) [inline docs/api.md]: Much more descriptive, thanks.

**Review by a-type — APPROVED** [review]

**Pull request merged**